### PR TITLE
feat: Condense casing spawns at shooting ranges

### PR DESCRIPTION
**Pull request by RobbieNeko**
## Purpose of change

Having all those casings littered all over the ground is very silly, and does not look good. Plus, it's pretty unrealistic that they wouldn't have some sort of receptacle for them.

## Describe the solution

- Adds two trashcans to the shooting range that now hold most of the casing spawns
- Removed most of the loose casing spawns

## Describe alternatives you've considered

- Completely remove loose casing spawns

Nah, it's foreseeable that *some* small number of casings would be on the ground around the benches.

## Testing

It loads

## Additional context

Presumably breaks No-Hope a little more, but what doesn't these days

image

### CODE REVIEW

File: data/json/mapgen/shooting_range.json [hunk 1]
```diff
@@ -32,9 +33,9 @@
         { "chance": 25, "item": "office", "x": [ 3, 4 ], "y": 4 }
       ],
       "items": {
-        "w": { "item": "casings", "chance": 60, "repeat": 7 },
-        "a": { "item": "casings", "chance": 50, "repeat": 5 },
-        "#": { "item": "casings", "chance": 40, "repeat": 3 },
+        "w": { "item": "casings", "chance": 60, "repeat": 4 },
+        "%": { "item": "casings", "chance": 100, "repeat": 200 },
```
Comment (chaosvolt) [inline data/json/mapgen/shooting_range.json:37]: Maybe toned down just a lil, but more relevant is giving it a lil randomness.
```suggestion
        "%": { "item": "casings", "chance": 50, "repeat": 100 },
```

Comment (RobbieNeko) [reply data/json/mapgen/shooting_range.json:37]: Having tested it with 100 before I made it 200, it felt a bit lackluster personally. Yes, there's two cans, but per-can due to the fact that there are so many darn casings it's very spread out.

image

Comment (RobbieNeko) [reply data/json/mapgen/shooting_range.json:37]: Perhaps a compromise of 150, with some randomness to it?

Comment (chaosvolt) [reply data/json/mapgen/shooting_range.json:37]: Sure, that works. :D

Comment (RobbieNeko) [reply data/json/mapgen/shooting_range.json:37]: Alright. 150, 66 chance sound good?

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**